tests/frontends/test_pytorch_ops.py

```diff
@@ -53,6 +53,11 @@ def is_tensor_param(x):
     return False
 
 
+def _make_sens(o, _sens=3.21):
```

Comment:
Why the _ prefix on the sens argument?

Add a comment to explain why we're setting sens to 3.21 instead of 1.0.